client/web/src/components/AdvancedOptions/index.js

```diff
@@ -185,6 +191,16 @@ const AdvancedOptions = ({
           handleChange={handleSearchChange}
           handleTooltipClick={() => {}}
         />
+        <OptionSwitch
+          checked={useEchoCancellation}
+          name={'Enable Echo Cancellation'}
```

Comment:
change to "Open Speaker" for more clear message

Reply:
Done